pkg/handlers/system_intake_action.go

```diff
@@ -0,0 +1,86 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/google/uuid"
+	"github.com/gorilla/mux"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+type createSystemIntakeAction func(context.Context, uuid.UUID, *models.ActionType) error
```

Comment:
Ohhhh, wait a minute. This is "create an action that affects a system intake", not "this action creates a system intake"...?!? Okay, is less problematic w/r/t what I said about ID generation above.
Naming things is one of the hardest things! 😱

Reply:
Right, I wish I had a better naming convention to explain "this function creates an action that submits a system intake" rather than "this function is an action that creates a system intake"